bluemira/builders/tf_coils.py

```diff
@@ -467,15 +467,33 @@ def __init__(
         n_rip_points: int = 100,
         n_koz_points: int = 100,
         ripple_selector: Optional[RipplePointSelector] = None,
+        separatrix: Optional[BluemiraWire] = None,
     ):
         self.parameterisation = parameterisation
         self.params = make_parameter_frame(params, RippleConstrainedLengthGOPParams)
-        self.separatrix = separatrix
         self.wp_cross_section = wp_cross_section
         self.algorithm = algorithm
         self.opt_parameters = opt_parameters
         self.opt_conditions = opt_conditions
-
+        if ripple_wire is not None:
+            self.ripple_wire = ripple_wire
+            if separatrix is not None:
+                warnings.warn(
+                    "Argument 'separatrix' is deprecated, "
+                    "argument 'ripple_wire' is used instead.",
+                    category=DeprecationWarning,
+                    stacklevel=2,
+                )
+        elif separatrix is not None:
+            warnings.warn(
+                "Argument 'separatrix' is deprecated. "
+                "Please use argument 'ripple_wire' instead.",
+                category=DeprecationWarning,
+                stacklevel=2,
+            )
+            self.ripple_wire = separatrix
+        else:
+            raise ValueError("No value for ripple_wire found!")
```

Comment:
I think we can shorten this:
```suggestion
        if separatrix is not None:
            if ripple_wire is None:
                ripple_wire = separatrix
            warnings.warn(
                    "Argument 'separatrix' is deprecated, "
                    "argument 'ripple_wire' is used instead.",
                    category=DeprecationWarning,
                    stacklevel=2,
                )
        elif ripple_wire is None:
            raise ValueError("No value for ripple_wire found!")
        self.ripple_wire = ripple_wire      
```